packages/calypso-build/webpack/sass.js

```diff
@@ -69,12 +73,15 @@ module.exports.loader = ( {
  * @returns {object[]}                 styling relevant webpack plugin objects
  */
 module.exports.plugins = ( { chunkFilename, filename, minify } ) => [
-	new MiniCssExtractPluginWithRTL( {
+	new MiniCssExtractPlugin( {
 		chunkFilename,
 		filename,
 		ignoreOrder: true, // suppress conflicting order warnings from mini-css-extract-plugin
-		rtlEnabled: true,
+		attributes: {
+			'data-webpack': true,
+		},
 	} ),
+	new MiniCSSWithRTLPlugin(),
 	new WebpackRTLPlugin( {
```

Comment:
Makes me think if it's a good idea to merge `WebpackRTLPlugin` and `MiniCSSWithRTLPlugin` into one plugin that does the entire RTL setup? Or did it happen already in the upstream branch?

Reply:
That's a good idea. Once the migration is over there are many refactor opportunities for `WebpackRTLPlugin`, including merging it with the `MiniCSSWithRTLPlugin` as you mentioned.